packages/flame/lib/src/components/sprite_animation_group_component.dart

```diff
@@ -79,6 +100,29 @@ class SpriteAnimationGroupComponent<T> extends PositionComponent
 
   SpriteAnimation? get animation => animations?[current];
 
+  /// Returns the current group state.
+  T? get current => _current;
+
+  /// The the group state to given state.
+  ///
+  /// Will update [size] if [autoResize] is true.
+  set current(T? value) {
+    _current = value;
+    _resizeToSprite();
+  }
+
+  /// Returns current value of auto resize flag.
+  bool get autoResize => _autoResize;
+
+  /// Sets the given value of autoResize flag.
+  ///
+  /// Will update the [size] to fit srcSize of
+  /// current animation sprite if set to  true.
```

Comment:
The dartdocs should only linebreak mid sentence when then reach 80 in line length.

Reply:
Done.